plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/ConclaveTask.kt

```diff
@@ -94,9 +94,9 @@ abstract class ConclaveTask : DefaultTask() {
         }
     }
 
-    fun commandWithOutput(vararg command: Any): String {
+    fun commandWithOutput(vararg command: String, workingDir: String? = null): String {
```

Comment:
workingDir should be a `Path`.

Reply:
I'll leave this to another PR then. Maybe it is also a good idea to change the parameter of the other functions called in the call stack